addOns/importurls/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
+## [8] - 2020-12-15
 ### Changed
 - Update minimum ZAP version to 2.9.0.
```

Comment:
2.10?

Reply:
Arg, yeah, missed that ;)

Reply:
Done

Reply:
Actually, dont know why I included that one, have now removed it...